### DISCUSSION TITLE
2022-30: 如何维护一个开源项目

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#370] [Announcements]
# 2022-30: 如何维护一个开源项目

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-30/

**Comment by STRRL** [comment]
> 开源项目的发展也大体遵循着强者恒强的马太定律：一个特定的垂直领域往往只能容下一到两个广泛使用的项目。

> 如果我们不能够比现有项目做的更好，不如直接放弃加入现有项目。

太同意啦!

**Comment by Xuanwo** [comment]
[chrono](https://github.com/chronotope/chrono) 也迎来了新的维护者： https://github.com/chronotope/chrono/releases/tag/v0.4.20-rc.1 ，真好啊

**Reply by ttys3** [reply]
chrono 再不更新都要死了, 很多项目早前都直接迁移到 time 了

**Comment by noirgif** [comment]
最近 chroumium 正在退役 manifest v2，推行在一定程度上让广告和追踪更难控制的v3.我就在怀疑：各大浏览器放弃开发内核而加入chroumium伞下是对的吗？
